plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/server/ServerSettingsWizardFragment.java

```diff
@@ -521,4 +526,9 @@ public void setComplete(boolean complete) {
 		super.setComplete(complete);
 	}
 
+	@Override
+	public boolean isComplete() {
+		return uiHook != null && !uiHook.isDisposed() && super.isComplete();
+	}
```

Comment:
This code seems correct to me.  I'm not sure if it's the 'proper' solution, but the available API leaves no other options really.